.github/settings.yml

```diff
@@ -0,0 +1,46 @@
+repository:
+  name: holo-envoy
+  description: Server that facilitates communication between hosted Agents and a Host's Holochain Conductor
+  topics: holo, service-logging
+  private: false
+  has_issues: true
+  has_projects: false
+  has_wiki: false
+  default_branch: master
```

Comment:
Is this correct?

Reply:
If you mean the default branch being `master`, yes, that is on purpose.  I've never understood the value of making the default `develop`.

It seems to me that making `develop` the default is only useful before a project has a stable release.  When I go to a project, I expect to see the best version of it, not the most up to date version.